sub-packages/bionemo-esm2/tests/bionemo/esm2/scripts/test_infer_esm2.py

```diff
@@ -144,11 +139,21 @@ def test_esm2_fine_tune_data_module_val_dataloader(data_module):
 
 
 @pytest.mark.parametrize("precision", ["fp32", "bf16-mixed"])
+@pytest.mark.parametrize("checkpoint_path", [esm2_3b_checkpoint_path, esm2_650m_checkpoint_path])
```

Comment:
Do we really need the 3b test?

Reply:
oh, looks like I am not even using that in the test